src/query/interpreter.hpp

```diff
@@ -247,6 +248,14 @@ class Interpreter final {
     std::optional<std::string> db;
   };
 
+#ifdef MG_ENTERPRISE
+  struct RouteResult {
+    int ttl{300};
```

Comment:
What was the reason for setting this up to 300?

Reply:
I saw default on neo4j's side, seems reasonable